.gitignore

```diff
@@ -3,3 +3,4 @@
 .idea
 _cluster-up/
 _kubevirt/
+_schemas/main/*
```

Comment:
Change that to `_schemas/`?

Reply:
@0xFelix  I can add a comment here echoing what I wrote in the commit but I only wanted to ignore the `main` branch in the cache as it's likely to change so often. For the more stable release branches I thought we could continue using the cache as things are less likely to change. Thoughts?

Reply:
Makes sense, I missed the comment in the commit message, sorry.